index.js

```diff
@@ -327,7 +327,7 @@ function formatAnswer(reportData, url, scan, options) {
     scan.score = newScore;
   }
 
-  if (options.nagios != undefined) {
+  if (options.nagios !== undefined) {
```

Comment:
as it turns out, `0` is a valid value :)    undefined should be ALWAYS  '==='  NEVER '=='!   Good catch.